src/components/Appbar.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { CogIcon, ChevronLeftIcon, MicrophoneIcon } from '@heroicons/react/solid';
+
+const Appbar = () => (
+  <div className="flex items-center h-14 px-2 bg-pink-600">
+    <NavLink className="basis-1/3 flex" to="/">
```

Comment:
- [ ] The Home page should have a list of categories that could be filtered using given parameters, e.g. category name. Kudos on following the design guidelines and creating the same layout, however I cannot find the search functionality. 
Kindly add search field that will filter the results by stock name.